src/main/resources/assets/minecolonies/gui/townhall/windowtownhall.xml

```diff
@@ -2,71 +2,61 @@
 
     <image source="minecolonies:textures/gui/townhall_book.png" pos="75 0" size="374 243"/>
 
```

Comment:
We have six ribbon textures, can you use them all equally instead one many times?
Plus you need to check extended versions down matches the IDs here